management_api_app/services/authentication.py

```diff
@@ -0,0 +1,120 @@
+from fastapi import Request, Depends, HTTPException, status
+from fastapi.security import OAuth2AuthorizationCodeBearer, HTTPBearer
+from typing import List, Optional
+from core import config
+import base64
+import jwt
+import requests
+import rsa
+
+class User:
+    def __init__(self, decoded_token: dict):
+        self._claims = decoded_token
+
+    @property
+    def id(self):
+        return self._get_claim('oid')
+
+    @property
+    def name(self):
+        return self._get_claim('name')
+
+    @property
+    def email(self):
+        return self._get_claim('email')
+
+    @property
+    def roles(self):
+        return self._get_claim('roles')
+
+    def _get_claim(self, claim: str):
```

Comment:
_claims is a dict an dict has a .get(blah, default). Instead of checking if it exists you should use that for more pythonic idiom